middleware/sisyphus-rocketmq-ons/src/main/kotlin/com/bybutter/sisyphus/middleware/rocketmq/DefaultRocketTemplateFactory.kt

```diff
@@ -0,0 +1,60 @@
+package com.bybutter.sisyphus.middleware.rocketmq
+
+import com.aliyun.openservices.ons.api.Consumer
+import com.aliyun.openservices.ons.api.ONSFactory
+import com.aliyun.openservices.ons.api.Producer
+import com.aliyun.openservices.ons.api.PropertyKeyConst
+import com.aliyun.openservices.ons.api.PullConsumer
+import com.aliyun.openservices.ons.api.batch.BatchConsumer
+import com.aliyun.openservices.ons.api.order.OrderConsumer
+import com.aliyun.openservices.ons.api.order.OrderProducer
+import com.aliyun.openservices.ons.api.transaction.LocalTransactionChecker
+import com.aliyun.openservices.ons.api.transaction.TransactionProducer
+import java.util.Properties
+
+class DefaultRocketTemplateFactory : RocketTemplateFactory {
+    private var properties: Properties? = null
+
+    override fun getMQProperties(property: RocketMQProperty): Properties {
+        if (properties != null) {
+            return properties!!
```

Comment:
```suggestion
            return properties!!
```
用properties?.let{}优化下代码把!!去掉